app/apollo/test/channel.fga.spec.js

```diff
@@ -0,0 +1,666 @@
+/**
+ * Copyright 2020 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
As this is a brand new test file, it would be good to include a descriptive comment explaining its overall purpose, e.g.:
```
/*
Fine-Grained-Authorization (FGA) allows the auth model to determine access to resources based on the name or id of the resource, and on the action being taken (e.g. read vs edit).
This test suite provides validation of FGA behavior for auth models that support it by verifying that users with different FGA permissions are correctly allowed or restricted.
Other test suites will validate behavior based on having admin permissions (or lack thereof) independently.
*/
```

Reply:
Added this. Should it be within the license comment or its own under the license comment?

Reply:
It's not part of the License, so should be separate.

Reply:
Comment separated from license